docs/content/staking/rewards.mdx

```diff
@@ -4,10 +4,25 @@ sidebar_title: Rewards
 description: How the Flow protocol pays rewards to stakers
 ---
 
-# Rewards
+## Rewards
 
-Rewards are paid out at the end of each epoch to all the users that have tokens staked.
+Rewards are paid out at the end of each [epoch](/staking/#epochs) to all the users that have tokens staked.
 Rewards are considered to be unlocked and liquid, so the user can withdraw their rewards
 at any time.
 
 More information about the rewards calculation and schedule will be provided in the near future.
+
+If you staked using a 3rd party staking provider such as Kraken, Blocto or Finoa, please read [our guides](/staking/custody-providers/) for more information about how
+to claim and use your rewards via your chosen provider.
+
+## Rewards Calculation
```

Comment:
Have you gotten confirmation from Layne or Roham that this info is allowed to be shared?